src/modules/logger/logger.cpp

```diff
@@ -476,6 +476,7 @@ bool Logger::copy_if_updated_multi(LoggerSubscription &sub, int multi_instance,
 void Logger::add_default_topics()
 {
 	add_topic("vehicle_attitude", 10);
+	add_topic("vehicle_attitude_groundtruth", 10);
```

Comment:
I think that would be better than having to use a custom topic list on the SD card.